### DISCUSSION TITLE
Why Speed selector instead of Steps slider when there are now presets?

### DISCUSSION THREAD

**Discussion by realmalikov** [lllyasviel/Fooocus#2929] [Q&A] [answered]
Does anyone else besides me find the “Performance” choices a bit confusing and pointless, especially since there are preset settings, some of which have the same name as the Speed ​​option?  This becomes particularly problematic as more and more speed options are added.  First came LCM with the corresponding “extreme speed” setting, then Lightning with the “lightning” speed setting, and now you could even add a HyperSD setting.  But all of these options have presets too, right?  Wouldn't a simple Steps slider combined with a preset selector be simpler and more efficient?

**Comment by Rogala** [comment, marked as answer]
Fooocus is designed for ordinary people, the interface has a minimum of settings and they are all easy to understand, for more geeks who want to control finer settings there is stable-diffusion-webui, etc.

**Reply by ILikeToasters** [reply]
Or just go into debug settings.

**Comment by mashb1t** [comment]
@realmalikov you can already set the steps in the developer debug mode.
A slider in this case is not sufficient, as performance modes also load LoRAs and other things in the background. As already pointed out by @Rogala, Fooocus is designed for both light and heavy users, and light users do not need to be confronted with a granular step slider by default.

Fun fact: Performance and preset Hyper-SD is already fully functional in PR https://github.com/lllyasviel/Fooocus/pull/2812
